### PR TITLE
Climber changes during Sac Day 1

### PR DESCRIPTION
**Pull request by kevinzwang**
- slow climb
- added Lily's fix to KeepClimber command problem

### CODE REVIEW

File: Robot2019/src/main/java/frc/robot/commands/NormalDrive.java [hunk 1]
```diff
@@ -7,26 +7,11 @@
 
 package frc.robot.commands;
 
-import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.InstantCommand;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
-public class NormalDrive extends Command {
+public class NormalDrive extends InstantCommand {
```
Comment (brettle) [inline Robot2019/src/main/java/frc/robot/commands/NormalDrive.java]: This should really be renamed something like ToggleCharDrive.

**Review by brettle — CHANGES_REQUESTED** [review]

**Comment by kevinzwang** [comment]
slow climb was never used and is now obsolete because we have a software limit to our climber elevator

**Comment by kevinzwang** [comment]
but we should make these aesthetic changes in another pull request

**Review by brettle — APPROVED** [review]

**Comment by brettle** [comment]
@kevinzwang, I've approved, so you should be able to merge now if you feel it's ready.

**Pull request merged**